cmd/kured/main.go

```diff
@@ -437,19 +437,20 @@ func drain(client *kubernetes.Clientset, node *v1.Node) {
 			log.Fatalf("Error cordonning %s: %v", nodename, err)
 		}
 		log.Errorf("Error cordonning %s: %v, continuing with reboot anyway", nodename, err)
-		return
+		return err
 	}
 
 	if err := kubectldrain.RunNodeDrain(drainer, nodename); err != nil {
 		if !forceReboot {
 			log.Fatalf("Error draining %s: %v", nodename, err)
 		}
 		log.Errorf("Error draining %s: %v, continuing with reboot anyway", nodename, err)
```

Comment:
is continue with reboot anyways true anymore? looking for consuming function.

Reply:
yeah I think this should just say error drain.

Reply:
yeah you're right some more grunt work was needed here